src/extensions/replay/sessionrecording.ts

```diff
@@ -1269,17 +1269,15 @@ export class SessionRecording {
             return
         }
 
+        // we can't flush the buffer here since someone might be starting on a blocked page,
+        // and we need to be sure that we don't record that page
+        // so we might not get the below custom event but events will report the paused status
+        // which will allow debugging of sessions that start on blocked pages
         this._urlBlocked = true
-        document?.body?.classList?.add('ph-no-capture')
```

Comment:
this snuck past me in the initial implemention. naughty paul
this doesn't just add belt and braces for not capturing the DOM, it stops other capture like autocapture too... no bueno